### DISCUSSION TITLE
How to show the marker for leafmap searching

### DISCUSSION THREAD

**Discussion by zxdawn** [opengeos/leafmap#471] [Q&A] [answered]
I see that when I search a location using lat, lon with foliumap (import leafmap.foliumap as leafmap), it will jump to the scene and show the marker. Is it possible to show the marker with the default leafmap (import leafmap)?

![image](https://github.com/opengeos/leafmap/assets/30388627/45701131-1cf3-4db1-ac94-eacb9ed9c21c)

**Comment by giswqs** [comment, marked as answer]
Like this?

```python
import leafmap
m = leafmap.Map(search_control=False)
url = 'https://nominatim.openstreetmap.org/search?format=json&q={s}'
m.add_search_control(url, zoom=10)
m
```
![image](https://github.com/opengeos/leafmap/assets/5016453/e21675b4-17f2-4027-835e-e96c33881395)

**Reply by zxdawn** [reply]
Yes, thanks for the tip.
